docs/configuration/config-file-reference.md

```diff
@@ -2438,6 +2438,37 @@ grpc_client_config:
   # Skip validating server certificate.
   # CLI flag: -querier.frontend-client.tls-insecure-skip-verify
   [tls_insecure_skip_verify: <boolean> | default = false]
+
+memory_limiter:
+  # Enable memory-limiter.
+  # CLI flag: -querier.memory-limiter.enabled
+  [enabled: <boolean> | default = false]
+
+  # Specifies whether to fail when the heap is above the limit. If fail fast is
```

Comment:
`when the heap is above the limit` this is red state, right? Can you mention it in the doc also